db/migrate/20161030034120_change_status_type.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeStatusType < ActiveRecord::Migration
+  def change
+    change_column(:notifications, :status, :integer)
```

Comment:
Changing `status` to an integer to manage it with `enum`. Unnecessary DB size when stored in `string`, it's better to store it as an `integer`.